### PR TITLE
Fix for Perkins loan caps, unsubsidized graduate loan error message

### PR DESCRIPTION
**Pull request by mistergone**
This PR implements two fixes:
- Updates to `student-debt-calc` v2.5.11, which fixes errors with Perkins Loan caps.
- Fixes error content for graduate unsubsidized loans
## Changes
- Update to `student-debt-calc` v2.5.11
- Moves calculation of caps and updating of content regarding caps to `financialView.updateViewWithProgram()` which is much better timing
## Testing
- Passes unit tests!
## Review
- @marteki @higs4281 - Pull it down, `npm install`, `gulp`, and have a good time!
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [x] Passes all existing automated tests
- [x] New functions include new tests
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged
- [x] Visually tested in supported browsers and devices
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
## Animated GIF

**When I notice how poorly I coded something in the recent past:**

spock-dazzling-logic

### CODE REVIEW

**Comment by saintsoup52** [comment]
Does this pull request deal with perkins loans that exceed cost of attendance as well?

**Comment by mistergone** [comment]
@saintsoup52 - Yes

File: src/disclosures/js/index.js [hunk 1]
```diff
@@ -63,7 +63,6 @@ var app = {
             } );
         }
         // set financial caps based on data
-        financialView.setCaps( getFinancial.values() );
```
Comment (marteki) [inline src/disclosures/js/index.js:67]: This line needs to remain in order for the About This Tool page to display loan and grant caps.

**Comment by marteki** [comment]
On focusout, the Grad PLUS loan over cost of attendance error is flashing off screen before it can be read.

File: npm-shrinkwrap.json [hunk 2]
```diff
@@ -351,9 +351,9 @@
       "resolved": "https://registry.npmjs.org/bluebird/-/bluebird-3.4.1.tgz"
     },
     "bn.js": {
-      "version": "4.11.6",
+      "version": "4.11.5",
```
Comment (marteki) [inline npm-shrinkwrap.json:354]: In your reducing of merge conflicts, you might have downgraded a lot of dependencies in our shrinkwrapping? I just upgraded a bunch of our NPM packages.

File: src/disclosures/js/index.js (hunk 1 above)
Comment (mistergone) [inline src/disclosures/js/index.js:67]: I replaced this, and I'll push a commit up.

**Comment by mistergone** [comment]
> On focusout, the Grad PLUS loan over cost of attendance error is flashing off screen before it can be read.

@marteki - Was this previously addressed? It's possible a merge messed it up.

**Comment by marteki** [comment]
Focusout: Yeah. I just had a PR merged today to address it. 😞  https://github.com/cfpb/college-costs/pull/267

**Comment by mistergone** [comment]
@marteki - I can't account for that. Your changes from #267 are still in my PR, and I don't immediately see what could have broken them.

File: npm-shrinkwrap.json (hunk 2 above)
Comment (mistergone) [inline npm-shrinkwrap.json:354]: Oh, I might have. There were conflicts, so I regenerated the file. I don't know why that wouldn't keep the dependency tree...

**Comment by mistergone** [comment]
@marteki - The latest commit should fix your error messages not-appearing bug. 😄

**Comment by marteki** [comment]
It does! 🙌 

:+1:

**Comment by higs4281** [comment]
👍 perkins caps show correctly for grad and undergrad perkins schools.

> Does this pull request deal with perkins loans that exceed cost of attendance as well?

Working for me:

perkins_error_msgs

**Comment by mistergone** [comment]
A commit was also pushed to patch a bug in "About this page"

**Pull request merged**